test/table-test.mjs

```diff
@@ -35,178 +35,261 @@ const baseOperations = {
   }
 };
 
-it("makeQueryTemplate null table", () => {
-  const source = {};
-  assert.strictEqual(makeQueryTemplate(EMPTY_TABLE_DATA.operations, source), undefined);
-});
-
-it("makeQueryTemplate no selected columns", () => {
-  const source = {name: "db", dialect: "postgres"};
-  const operationsColumnsNull = {...baseOperations, select: {columns: null}};
-  assert.strictEqual(makeQueryTemplate(operationsColumnsNull, source), undefined);
-  const operationsColumnsEmpty = {...baseOperations, select: {columns: []}};
-  assert.strictEqual(makeQueryTemplate(operationsColumnsEmpty, source), undefined);
-});
-
-it("makeQueryTemplate invalid filter operation", () => {
-  const source = {name: "db", dialect: "postgres"};
-  const invalidFilters = [
-    {
-      type: "n",
-      operands: [
-        {type: "column", value: "col1"},
-        {type: "primitive", value: "val1"}
-      ]
-    },
-    {
-      type: "eq",
-      operands: [{type: "column", value: "col1"}]
-    },
-    {
-      type: "lt",
-      operands: [
-        {type: "column", value: "col1"},
-        {type: "primitive", value: "val1"},
-        {type: "primitive", value: "val2"}
-      ]
-    }
-  ];
-
-  invalidFilters.map((filter) => {
-    const operations = {
-      ...baseOperations,
-      filter: [filter]
-    };
-    assert.throws(() => makeQueryTemplate(operations, source), /Invalid filter operation/);
+describe("makeQueryTemplate", () => {
```

Comment:
the only change i made here was to wrap the `makeQueryTemplate` tests in a `describe` – if you check "Hide whitespace", this diff will look much better 😄